<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="c0016390" failures="6" tests="9" name="Rebill Test" time="537.29" errors="0" timestamp="2020-12-23T21:54:26 UTC">
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod2" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod2() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod4" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod4() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod4 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod3" time="0.0">
    <failure type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] rerate.rerateTestNgSlow.dataProviderMethod3() returned a null value
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- testMethod3 -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="44.591">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:987)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:802)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:636)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="43.841">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:987)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:802)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:636)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="132.07"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="43.429">
    <failure type="java.lang.AssertionError" message="Out of Scope">
      <![CDATA[java.lang.AssertionError: Out of Scope
at rerate.rerateTestNgSlow.firstPage(rerateTestNgSlow.java:987)
at rerate.rerateTestNgSlow.doWork(rerateTestNgSlow.java:802)
at rerate.rerateTestNgSlow.testMethod(rerateTestNgSlow.java:636)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- testMethod -->
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="134.803"/>
  <testcase classname="rerate.rerateTestNgSlow" name="testMethod" time="133.908"/>
</testsuite> <!-- Rebill Test -->
